lgc/elfLinker/FetchShader.cpp

```diff
@@ -40,15 +40,22 @@
 using namespace lgc;
 using namespace llvm;
 
+static constexpr uint32_t LsHsSysValueMergedWaveInfo = 3;
```

Comment:
nit: no need for `static`, it's already implied by `constexpr` which gives it internal linkage